core/src/main/kotlin/in/specmatic/core/Feature.kt

```diff
@@ -365,19 +365,13 @@ data class Feature(
             val negativeTestScenarios =
                 negativeScenario.generateTestScenarios(flagsBased, testVariables, testBaseURLs)
 
-            negativeTestScenarios.filterNot { negativeTestScenarioR ->
+            val filtered = negativeTestScenarios.filterNot { negativeTestScenarioR ->
```

Comment:
-  `negativeTestScenarioR` is essentially a temporary variable used to hold each individual negative test scenario as it's being evaluated by the `filterNot` function. 
- This will be renamed to `currentNegativeTestScenario`.  So the name indicates that this is the current negative test scenario being evaluated in the filter operation.

Reply:
Fixed in : [Renamed variable 'negativeTestScenarioR' to 'currentNegativeTestScena…](https://github.com/znsio/specmatic/pull/1082/commits/c0793823051441f79568451ee8e85e9d7bd588cd)